docs/guides/browse_output.md

```diff
@@ -0,0 +1,36 @@
+!!! important "Quick Look"
+    On the job screen on floydhub.com:
+
+    ![Download Output](../img/browse_output_partial.jpg)
+
+    Or from the CLI:
+    ```
+    $ floyd output <username>/<project_name>/<run_number>
```

Comment:
This will work, but for consistency, this should be `floyd output <username>/projects/<project_name>/<run_number>`

Reply:
Good catch. Fixed